nservicebus/licensing/license-management.md

```diff
@@ -110,3 +110,22 @@ As a developer you can specify the license to use in your configuration code:
 
 <!-- import License -->
 
+## Where does NServiceBus look for a license?
+
+If you are experiencing an issue with licensing, it can be helpful to know where NServiceBus is looking, and in what order. An expired license in `HKEY_CURRENT_USER`, for example, will overrule a valid license in `HKEY_LOCAL_MACHINE`. Note that these vary somewhat in older versions.
+
+In order to find the license, NServiceBus will examine:
+
+* License XML defined by `NServiceBus/License` appSetting
+* File path configured through `NServiceBus/LicensePath` appSetting
+* File located at `<BaseDirectory>\NServiceBus\License.xml`
```

Comment:
so by `<BaseDirectory>` do u mean `Environment.CurrentDirectory` or the directory of the current running exe/dll? This often makes a difference in the context of windows services.